actions/userinfo.js

```diff
@@ -22,38 +223,40 @@ function submitUserInfoBegin() {
 function submitUserInfoSuccess(data) {
   return {
     type: SUBMIT_USER_INFO_SUCCESS,
-    data,
+    data
   };
 }
 export function getUserId() {
-  return (dispatch) => {
+  return dispatch => {
     dispatch(submitUserInfoBegin());
-    return local.get('initialQuestions')
-    .then((resp) => {
-      if (resp.error) {
-        return dispatch(error(resp.error));
-      }
-      return dispatch(sendUserId(resp.data.user.id));
-    })
-    .catch((err) => {
-      return dispatch(error(err));
-    });
+    return local
+      .get('initialQuestions')
+      .then(resp => {
+        if (resp.error) {
+          return dispatch(error(resp.error));
```

Comment:
if you `throw` the error here, your catch can handle this for you.

Reply:
check